spec/loadbalancer/README.md

```diff
@@ -1,14 +1,10 @@
-## EC Load Balancer Spec
+## EC Gateway Load Balancer Spec
```

Comment:
@ramaraosrikakulapu please adding docker build badge on the top of the readme as follows-

```![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/enterpriseconnect/loadbalanceri)```

Reply:
Docker build status added